snappy_wrappers/wrappers/somatic_variants_checking/wrapper.py

```diff
@@ -39,8 +40,8 @@
 # Compute MD5 sums of logs
 shell(
     r"""
-md5sum {snakemake.log.log} > {snakemake.log.log_md5}
-md5sum {snakemake.log.conda_list} > {snakemake.log.conda_list_md5}
-md5sum {snakemake.log.conda_info} > {snakemake.log.conda_info_md5}
+md5sum $(basename {snakemake.log.log}) > {snakemake.log.log_md5}
```

Comment:
I think that's wrong. The shell won't find `$(basename {snakemake.log.log})`. What you need to do to avoid having the full path in the md5 checksum is, to go to the directory of `{snakemake.log.log}` and then issue the checksum command on the base name only